app/models/course/assessment.rb

```diff
@@ -9,7 +9,8 @@ class Course::Assessment < ApplicationRecord
 
   # Concern must be included below acts_as_lesson_plan_item to override #can_user_start?
   include Course::Assessment::TodoConcern
-  include Course::ReminderConcern
+  include Course::OpeningReminderConcern
```

Comment:
I think you can join the opening and closing together into 1 concern if you prefer as well.